lib/src/metta/runner/environment.rs

```diff
@@ -124,6 +126,17 @@ impl EnvBuilder {
         self
     }
 
+    /// Configures the environment to create a config directory with default config files, if no directory is found
+    ///
+    /// NOTE: If the config directory exists but some config files are missing, default files will not be created.
+    pub fn create_config_dir(mut self) -> Self {
+        self.create_cfg_dir = true;
+        if self.no_cfg_dir {
+            panic!("Fatal Error: create_config_dir is incompatible with set_no_config_dir");
+        }
+        self
+    }
+
     /// Configures the Environment not to load nor create any config files
     pub fn set_no_config_dir(mut self) -> Self {
```

Comment:
I would also check that `self.no_cfg_dir` value doesn't conflict with `self.create_cfg_dir` value.

Reply:
Added in [168231b](https://github.com/trueagi-io/hyperon-experimental/pull/478/commits/168231b61bdec8f1c80f368eceb86bd6669ea794)